### PR TITLE
BookList #Issue222 New pr

### LINKED ISSUE #222: Add a suggested Books for DSA webpage under More on nav bar
## **Is your feature request related to a problem? Please describe.**
Along with tons of online material, learners often search for list of books on to get a clearer idea of DSA
- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

## **Describe the solution you'd like**
A specified page designated for a list of books.
- A clear and concise description of what you want to happen.
Add a link to another webpage which will include list of suggested books and on hovering over each item on the list, we can add a few lines on how that particular book can help. The webpage will require a very simplistic design, just using HTML CSS containing list of books, I have 3 books in mind right now.

### PR DESCRIPTION
**Pull request by sohini3062**
# Description
New pr for adding suggested booklist webpage under More on the navigation bar using HTML and CSS. 

Fixes #222 

## List any dependencies that are required for this change 
The old pr number is
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
    - After :
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by sohini3062** [comment]
@harikesh409 The new Pr in reference to https://github.com/Algo-Phantoms/Algo-Phantoms-Frontend/pull/223

**Comment by harikesh409** [comment]
@sohini3062 resolve the conflicts and #208 is present which can be resolved.

**Comment by sohini3062** [comment]
Ok I am working on it

**Comment by sohini3062** [comment]
Resolved Conflicts.
Issue #208 fixed
**Before:**
before
**After:**
after2
after1

File: index.html [hunk 1]
```diff
@@ -89,7 +97,10 @@
                             <a class="dropdown-item" href="https://github.com/Algo-Phantoms" target="_blank">Contribute
                                 to Project</a>
 
-                            <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
+                            <a class="dropdown-item" href="practicedsa.html">DSA Problems</a> master
```
Comment (harikesh409) [inline index.html:100]: what is this additional master?

File: css/bookliststyle.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  .list--link:hover .list--link__description,
+  .list--link:focus .list--link__description {
+    display: block;
+  }
+}
+#footer {
+  margin-top:20vh;
+  padding: 100px 0px 40px 0px;
+  background-image: linear-gradient(130deg, #05272e 40%, #183a36 100%);
+  /* background-image:linear-gradient(130deg, #ffa159 10%, #ff39b4 51%, #ffa159 100%);    */
```
Comment (harikesh409) [inline css/bookliststyle.css:143]: Remove the comment if not using.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: css/bookliststyle.css (hunk 2 above)
Comment (sohini3062) [reply css/bookliststyle.css]: Ok done

**Comment by harikesh409** [comment]
@sohini3062 resolve conflicts.

**Comment by sohini3062** [comment]
i have resolved the conflicts.

**Review by harikesh409 — APPROVED** [review]

**Comment by sohini3062** [comment]
Is there anything else to be added ?

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**